test/integration/fapi-check-wrong-paths.int.c

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
Again - this should probably be in a separate commit, especially if it needs to be back-ported - just for the future.